usage/rest-client/pom.xml

```diff
@@ -114,14 +114,14 @@
             <groupId>org.apache.brooklyn</groupId>
             <artifactId>brooklyn-core</artifactId>
             <version>${project.version}</version>
+            <classifier>tests</classifier>
             <scope>test</scope>
         </dependency>
         <dependency>
             <groupId>org.apache.brooklyn</groupId>
             <artifactId>brooklyn-core</artifactId>
             <version>${project.version}</version>
-            <classifier>tests</classifier>
-            <scope>test</scope>
+            <scope>compile</scope>
```

Comment:
why do we need to add `<scope>compile</scope>`?

previously both dependencies, regular and `<classifier>tests</classifier>` were only brought in for `test` scope I think which lowers the footprint for someone using the REST client.  also the ordering of the regular classifier first is cleaner i think.